includes/multi-currency/client/blocks/currency-switcher.js

```diff
@@ -202,138 +203,159 @@ registerBlockType( 'woocommerce-payments/multi-currency-switcher', {
 
 		return (
 			<div { ...blockProps }>
-				<InspectorControls key="setting">
-					<PanelBody
-						title={ __(
-							'Multi-Currency settings',
-							'woocommerce-payments'
-						) }
-					>
-						<a
-							href={
-								'/wp-admin/admin.php?page=wc-settings&tab=wcpay_multi_currency'
-							}
-							target="_blank"
-							rel="noreferrer"
-						>
-							{ __(
-								'Adjust and edit your Multi-Currency settings',
-								'woocommerce-payments'
-							) }
-						</a>
-					</PanelBody>
-					<PanelBody title={ __( 'Layout', 'woocommerce-payments' ) }>
-						<CheckboxControl
-							label={ __(
-								'Display flags',
+				<WordPressComponentsContext.Provider value={ wp.components }>
+					<InspectorControls key="setting">
+						<PanelBody
+							title={ __(
+								'Multi-Currency settings',
 								'woocommerce-payments'
 							) }
-							checked={ flag }
-							onChange={ onChangeFlag }
-						/>
+						>
+							<a
+								href={
+									'/wp-admin/admin.php?page=wc-settings&tab=wcpay_multi_currency'
+								}
+								target="_blank"
+								rel="noreferrer"
+							>
+								{ __(
+									'Adjust and edit your Multi-Currency settings',
+									'woocommerce-payments'
+								) }
+							</a>
+						</PanelBody>
+						<PanelBody
+							title={ __( 'Layout', 'woocommerce-payments' ) }
+						>
+							<CheckboxControl
+								label={ __(
+									'Display flags',
+									'woocommerce-payments'
+								) }
+								checked={ flag }
+								onChange={ onChangeFlag }
+								__nextHasNoMarginBottom
```

Comment:
This additional prop in the WP components creates a slight difference in layout compared to the previous implementation. However, I don't think it's impactful. If nothing else, it's consistent with the other WP components.
![2025-07-02 14 57 14](https://github.com/user-attachments/assets/642dd330-af6e-4132-9186-a3eecc43ad47)